hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/CodecCodeGenerator.java

```diff
@@ -314,6 +314,8 @@ public void generateCodec(CodecModel codecModel, Template codecTemplate) {
             String fileName = codecModel.getClassName() + "." + codecModel.getLang().name().toLowerCase();
             if (codecModel.getLang() == Lang.PY) {
                 fileName = fileName.replaceAll("(.)(\\p{Upper})", "$1_$2").toLowerCase();
+            } else if (codecModel.getLang() == Lang.NODE) {
```

Comment:
@bilalyasar matter of taste and preference, but I prefer `switch` instead of `if - else-if` statements